src/changelog/comment_changes.rs

```diff
@@ -296,6 +301,12 @@ impl Logic {
               }
 
               if let Ok(oid) = oid {
+                if let Some(stop_at) = self.cli.stop_at {
+                  if stop_at == oid {
+                    break;
+                  }
+                }
```

Comment:
If the stop SHA is reached, the evaluation will be stopped.  In case there should be still commits to analyse as given by `-n`, the stop commit will override count settings and vice versa.